doc/mesh_filter/urdf/ur5_sensor.gazebo

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```

Comment:
Nothing in this file refers to UR. Also it's not just "sensor", but a camera model.
How about calling the file `kinect_camera.gazebo`?

why do you even specify the camera model in a separate file at all?

Reply:
Renamed this file